Terceira Iteração : Implementação e manutenção de features.

psg2 opened the pull request.

cjvg committed:
created gui and controllet scenarios for the feature: auto evaluation edit

cjvg committed:
created gui and controller scenarios for the feature: auto evaluation edit

Commit:
Added the Import concepts from spreadsheet feature, including controller and GUI scenarios for valid and invalid spreadsheet.

Commit:
New Feature : Manual Concept Input with controller and GUI scenarios

cjvg committed:
New Feature : Auto Evaluation Edit with controller and GUI scenarios

Commit:
Merge pull request #1 from psg2/master

New Feature : Manual Concept Input with controller and GUI scenarios

dansq committed:
Register new disciplines

arquivo .feature com a descrição da feature para cadastrar novas
disciplinas

jgsjv committed:
Merge pull request #2 from dansq/NewDiscipline_feature

Register new disciplines

Commit:
feature CadastrodeCriterio

igormatos committed:
Create AccessControl.feature

igormatos committed:
Merge pull request #2 from igormatos/igormatos-TA-1

Create AccessControl.feature

igormatos committed:
Create AccessControl.feature

imvm committed:
Novo feature de notificações. 4 cenários, dois de controlador e dois de GUI. Para cada, um de sucesso e um de insucesso. Os cenários de controlador descrevem a reação do sistema no registro de notas, podendo ou não guardar uma notificação. Os cenários de GUI descrevem a interface durante a ação de um usuário checando suas notificações.

Commit:
Merge pull request #2 from gaabs/master

1 and 2 merged pull request

Commit:
Merge pull request #1 from imvm/master

Novo feature de notificações. 4 cenários, dois de controlador e dois …

Commit:
cenarios sobre a feature de comparação entre avaliação e auto avaliação

diest committed:
Merge pull request #1 from igormatos/patch-3

Create AccessControl.feature

imvm committed:
Merge pull request #1 from ehammo/master

Comparação

ncrneto committed:
Revert "mudança de teste"

This reverts commit be74ea4b982d1bb6fbc34960f97caa592f34fdf0.

imvm committed:
Merge pull request #2 from jgsjv/dev

Dev

renatodms committed:
Estou fazendo Commit deste arquivo por causa de problemas que aconteceram no meu repositório. Tive que refazer o repositório e por isso estou enviando este arquivo (que foi enviado pra mim por Caio (cjvg) corretamente)

renatodms committed:
re commit - new feature 'registration of teachers', 2 Controller scenarios and 2 GUI scenarios

Commit:
Merge pull request #5 from renatodeyvson/master

New features: Registration of Teachers and Auto Evaluation Edit

dansq committed:
Merge pull request #1 from imvm/master

OK

ncrneto committed:
DocumentacaoAutoavaliacaoPrazo

Ducumentacao de definicao do prazo de autoavaliacao.

ncrneto committed:
AutoavaliacaoPrazo

Documentacao sobre definicao de prazo de autoavaliacao

dansq committed:
Merge pull request #2 from psg2/master

Merge feito
4 New Features : Manual Concept Input, Edit Auto Evaluation, Import Concepts from Spreadsheets and Registration of Teachers

diest committed:
Merge pull request #2 from ncrneto/autoavaliacaoDocumentacao

AutoavaliacaoPrazo

cjvg committed:
Merge pull request #1 from dansq/master

Pull request 7 >> 3

cjvg committed:
Merge pull request #2 from diest/master

Commits dos alunos 9, 10 e 12

diest committed:
Review AccessControl.feature

Elementos do cenário como botões não deveriam ser especificados desta maneira, pois ela isola as plataformas. É uma boa prática deixar os cenários aplicáveis a várias plataformas diferentes.

diest committed:
Review AutoavaliacaoPrazo.feature

A rigor, os cenários estão com passos longos os quais poderiam ser divididos em dois com 'And' / 'E'. Só faltaram os cenários de interface gráfica.

Commit:
AutoEvaluation tests - First changes

Commit:
AutoEvaluation tests - Second change

imvm committed:
Update Notifications.feature

imvm committed:
Synchronizing local repository

Commit:
Import Concept Step file created

Commit:
Created Import Concepts and necessary auxiliary classes. Modified the feature to include parameters.

dansq committed:
Test implementation for RegisterDiscipline.

Changes in .feature file following monitor's observations (removed ":" from Given, When, Then and And).

Created auxiliary .groovy files for the test.

dansq committed:
Merge remote-tracking branch 'origin/master'

Conflicts:
	test/cucumber/RegisterDiscipline.feature

Commit:
Passos de teste
Classes auxiliares
Adaptação dos cenários de feature

Commit:
Ajustes finais

Commit:
Remoção de ações em branco.

Commit:
Created Sheet and SheetBuilder and made small corrections to ImportConcepts and ConceptController.

renatodms committed:
Groovy de testes da Feature RegistrationOfTeachers

imvm committed:
Merge pull request #3 from renatodeyvson/master

Groovy de testes da Feature RegistrationOfTeachers

imvm committed:
Beggining notification feature test implementation

imvm committed:
Beggining notification feature test implementation

imvm committed:
Merge branch 'master' of https://github.com/imvm/TA

cjvg committed:
Merge pull request #3 from gaabs/master

Import Concepts from SpreedSheet Test + auxiliary classes

cjvg committed:
Changes for auto evaluation

cjvg committed:
Change for auto evaluation 2

cjvg committed:
Merge remote-tracking branch 'origin/master'

Commit:
Update EvaluationComparison.groovy

Commit:
Update EvaluationComparison.groovy

Commit:
Update ShowComparisonPage.groovy

Commit:
Update EvaluationComparison.groovy

Commit:
Update EvaluationComparison.groovy

Commit:
Update ManualConceptInput.feature

Commit:
Update ManualConceptInput.feature

Commit:
Update ManualConceptInput.feature

Commit:
Update ManualConceptInputSteps.groovy

Commit:
Update ManualInputPage.groovy

Commit:
Edited chromedriver file and gebconfig to make tests run.

Corrected a few errors on features, tests and auxiliary classes from ImportConcepts feature.

diest committed:
Correções

Deixei minha feature melhor apropriada para a implementação de seu teste.

Commit:
Update EvaluationComparison.groovy

dansq committed:
Merge pull request #4 from ehammo/master

Merge de testes

cjvg committed:
Merge pull request #4 from gaabs/master

Edited chromedriver file and gebconfig to make tests run.

imvm committed:
Implementacao testes de notificacao

diest committed:
Create GradeCalculator.groovy

Esta classe só tem o método "vazio" calculateGrade para não gerar erro nos testes.

diest committed:
Create FinalGradeController.groovy

Uma interface para o método do qual precisarei nos testes. Ainda preciso incrementar mais para os testes não comprometerem nenhuma funcionalidade.

diest committed:
Create FinalGradeSteps.groovy

Primeira parte da criação dos testes da feature

diest committed:
Update FinalGrade.feature

Mais edições para melhorar a parte dos testes

diest committed:
Mais correções

Corrigi mais imprecisões dos cenários

diest committed:
Update FinalGrade.feature

dansq committed:
Correção de typos/nomes de classes

diest committed:
Delete GradeCalculator.groovy

Não era necessário

dansq committed:
Merge remote-tracking branch 'origin/master'

diest committed:
Interfaces incrementadas

Adicionadas mais interfaces para que o funcionamento não seja prejudicado.

diest committed:
Update FinalGradeSteps.groovy

Não fiz testes de GUI porque não entendi como minha feature deve se comportar nas páginas ou as ações que ela deva praticar em cada passo.

diest committed:
Correção de FinalGradeSteps.groovy

Corrigi alguns erros básicos

renatodms committed:
Merge pull request #1 from dansq/master

Testes groovy

renatodms committed:
Merge branch 'imvm-master'

renatodms committed:
Merge https://github.com/renatodeyvson/TA

Commit:
Merge pull request #6 from cjvg/master

Rearranjo de envio

Commit:
Merge pull request #2 from renatodeyvson/master

3o fase

Commit:
Merge branch 'master' of https://github.com/psg2/TA into psg2-master

Commit:
Update GebConfig.groovy

Commit:
Update GebConfig.groovy

Commit:
Update GebConfig.groovy

Commit:
Merge pull request #3 from ehammo/master

Merged

Commit:
Merge pull request #4 from diest/master

Commits do aluno 10

Commit:
Corrected many compilation errors.

Tried everything, but couldn't make Tests back to work since merge.

Commit:
Commented whole class AlternativeControllerSpec. It had incompatible commands to our grails version.

Full commented 	accesscontrol and              	autoavaliacaoprazo.

corrected pedro duplicate steps

corrected renato duplicate steps

corrected caio duplicate steps

corrected maia duplicate steps and classes

corrected ian notifications feature and autoavaliacaoprazo

deleted	cadastrodecriterios feature (unknown format)

dansq committed:
Merge remote-tracking branch 'upstream/master'

imvm committed:
Implementação feature Notifications

dansq committed:
entrega features, correção controlador, views, e domain

Commit:
Merge pull request #7 from gaabs/master

Pegar fork mais atualizado

Commit:
Domain class "Concept" created.

Commit:
Concept views

dansq committed:
arquivo foi modificado na máquina virtual, voltando ao original

jgsjv committed:
Merge pull request #6 from dansq/master

Merge da entrega 21/10/2015

Commit:
Atualizando repositório

imvm committed:
Delete modules.xml

imvm committed:
Setting Travis build

Commit:
Feature de inserção de conceitos manuais nos critérios.

imvm committed:
Deleting configuration file

imvm committed:
Merge https://github.com/imvm/TA

Commit:
Implemented scenario: importing invalid spreadsheet (gui and controller)

Commit:
Merge pull request #6 from psg2/master

Merge 3 >> 4

Commit:
Domain

Commit:
Backup

dansq committed:
Merge pull request #7 from jgsjv/master

Merge pull request #6 from dansq/master

Commit:
merge

dansq committed:
Merge branch 'igormatos-master'

Commit:
commit module

Commit:
Ian Merge

Commit:
Merge branch 'imvm-master'

Commit:
Merge Dansq

Commit:
Merge branch 'dansq-master'

Commit:
Merge branch 'master' of https://github.com/dansq/TA into dansq-master

Conflicts:
	.idea/modules.xml
	grails-app/controllers/ta/ConceptController.groovy
	grails-app/controllers/ta/NotificationController.groovy
	grails-app/controllers/ta/StudentController.groovy
	grails-app/views/student/index.gsp
	test/cucumber/AccessControl.feature
	test/cucumber/AutoavaliacaoPrazo.feature
	test/cucumber/steps/FinalGradeSteps.groovy
	test/functional/pages/ShowNotificationsPage.groovy
	test/functional/steps/NotificationsTestDataAndOperations.groovy

Commit:
Merge branch 'dansq-master'

Commit:
Correção de Conflitos

Commit:
Merge gaabs

Commit:
Merge branch 'gaabs-master'

Commit:
Merge branch 'master' of https://github.com/gaabs/TA into gaabs-master

Conflicts:
	grails-app/controllers/ta/ConceptController.groovy
	grails-app/views/concept/index.gsp
	grails-app/views/concept/upload.gsp

Commit:
Merge branch 'gaabs-master'

Commit:
Correção de Conflitos numero 2

pauloborba commented:
pessoal, o pull de vocês não passou pelo travis! vocês não instalaram o travis nos seus forks? vocês não testaram ante de submeter?

psg2 commented:
Eu não tive como tesar direito, pois quando recebi os últimos pull requests
apareceram conflitos e muitos erros de compilação e quando finalmente os
resolvi percebi que a classe de domínio Student havia sido alterada e boa
parte dos testes anteriores já não passavam mais com isso, e eu fiquei com
medo da contribuição de membros que fizeram o trabalho certo ser perdida
caso eu não enviasse o pull request.
O sistema todo do TA já estava uma bagunça mesmo antes dessa iteração e
está difícil ter um trabalho conciso com um grupo de 12 pessoas, a
comunicação não está sendo muito boa.
Seria bom ter uma nova reunião com o grupo e o stakeholder para esclarecer
as coisas e deixar claro quem não tem intenção de participar, porque pra
reorganizar as ordens de pull request está ficando ruim também.

2015-10-23 14:18 GMT-03:00 Paulo Borba notifications@github.com:

> pessoal, o pull de vocês não passou pelo travis! vocês não instalaram o
> travis nos seus forks? vocês não testaram ante de submeter?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/pauloborba/TA/pull/16#issuecomment-150637514.

Commit:
Comentários em testes com duplicate steps.

imvm committed:
geb config file

imvm committed:
geb config

imvm committed:
geb config

imvm committed:
pull from fork

imvm committed:
test adjustments

Commit:
Ajustes no Travis, testes e controller'

Commit:
Correção import errado

Commit:
Importação de planilha ok. Falta apenas adicionar os dados importados no estudante, dado que eles está mal estruturado.

Commit:
Correção de testes

Commit:
Alteraçao no caso de arquivo 'csv', pois o plugin nao importa arquivos nesse formato.

Commit:
Concept update, modifications in domain Student and controller Student.
Small updates in Student views as well

Commit:
alteracoes no formato planilha

Commit:
Calculo de media dos criterios

Commit:
Merge branch 'master' of https://github.com/gaabs/TA

Commit:
Merge branch 'Testes' of https://github.com/psg2/TA

Commit:
Alteração na importação de planilhas.
Caso o aluno ou criterio da planilha não exista, ele é cadastrado.

Commit:
Implementacao de Testes

cjvg committed:
Merge pull request #6 from psg2/Testes

Testes

Commit:
Alterações na feature de importação de planilhas
Alteração na importação de planilhas.
Verificação de colunas válidas

Commit:
Adição de form para view e metodo para o submit

Commit:
Adição de cenários e implementação de testes

Commit:
Adição de métodos auxiliares para teste de controller e GUI

Commit:
Correção do caso de erro da GUI

cjvg committed:
Mudanças para feature, steps e student

Commit:
Alterações na feature de importação de planilhas
Alteração na importação de planilhas (conceptController)
Verificação de colunas válidas (sheetImporter)
Alteração nos testes de importação de planilhas

Commit:
Adicionei arquivos auxiliares para testes (planilhas padrão e defeituosas)
Correção e implementação de testes da importação de planilhas

Commit:
Merge branch 'Testes' of https://github.com/psg2/TA

Conflicts:
	test/cucumber/steps/ManualConceptInputSteps.groovy

cjvg committed:
shows auto evaluation

cjvg committed:
changes to update auto evaluation

Commit:
Alterações na importação de planilhas, correção e implementação de respectivos testes

Commit:
Leitura JQuery da página de inserção de conceitos para testes de GUI
Alguns ajutes para o Controller e views de Student

Commit:
Compare Feature

Commit:
Configuração da IDE para Windows
Mudança dos títulos das páginas
Teste GUI semi-completo

Commit:
Adicionei arquivos auxiliares para testes (planilhas padrão e defeituosas)
Correção e implementação de testes da importação de planilhas

Commit:
gebconfig windows

Commit:
Merge remote-tracking branch 'origin/master'

Conflicts:
	test/functional/GebConfig.groovy

Commit:
Compare Feature steps

Commit:
Steps changes

Commit:
ImportConcepts from spreadsheet GUI tests

Commit:
files from gui test (sheet import)

Commit:
Steps and .feature changes
Pages and views updates

Commit:
ImportConcepts from spreadsheet  tests

Commit:
coloring differences

Commit:
-unit

Commit:
Finalização dos 8 cenários de teste.

cjvg committed:
changed methods for auto evaluation and the page

Commit:
Correção e implementação de testes da importação de planilhas

Commit:
[completed #107523732] [completed #109086474] Finalização dos 8 testes da Feature Manual Concept Input

Commit:
Tentativa de corrigir falhas nos testes. Há um problema de alteração concorrente no DB ainda não resolvido, e, aparentemente, o Travis não consegue compilar o Excel Importer.

Commit:
travis verbose

Commit:
test updates

imvm committed:
updating branch

Commit:
Adição de css para página da feature

Commit:
Steps Comentado

Commit:
Merge pull request #7 from ehammo/Testes

Steps Comentado

Commit:
Merge pull request #14 from psg2/Testes

Merge com branch

Commit:
Para o travis

Commit:
Para o travis - commit 2

cjvg committed:
[109091068] controller test for auto evaluation

dansq committed:
adicionando novos cenários e testes para RegisterDiscipline

renatodms committed:
Cenários de atualização de dados de professores

renatodms committed:
Merge pull request #3 from psg2/master

atualizar

imvm committed:
travis fix

imvm committed:
merge renato

imvm committed:
removed duplicated steps

imvm committed:
geb fix

dansq committed:
merge

dansq committed:
merge

cjvg committed:
Merge branch 'master' of https://github.com/gaabs/TA into gaabs-master

# Conflicts:
#	grails-app/controllers/ta/StudentController.groovy

cjvg committed:
Merge branch 'gaabs-master'

dansq committed:
erge

dansq committed:
Merge branch 'ehammo-master'

imvm committed:
Adjusting Notification controller and domain classes

imvm committed:
Fixed incorrect name error

imvm committed:
Removed duplicated steps

imvm committed:
Organized steps file

imvm committed:
Removed duplicate step, organized steps file

imvm committed:
Removed unnecessary link

imvm committed:
[#107883384]Developing scenario test

imvm committed:
[#107883384]Updated feature file

imvm committed:
[#107883384]Updated Notification class attributes

imvm committed:
[#107883384]Updated feature file

imvm committed:
[#107883384]Developing scenario tests

imvm committed:
[#107883412]Developing scenario tests

imvm committed:
[#107883384]Developing scenario tests

imvm committed:
[#107883384][#107883412]Updating steps

imvm committed:
[#107883384][#107883412]Changed test student name and login

imvm committed:
[#107883384][#107883412]Changed test parameters

imvm committed:
[#107883384][#107883412]Changed test parameters

imvm committed:
[#107883384][#107883412]Changed test parameters

imvm committed:
Added ignore

imvm committed:
[#107883384][#107883412]Developing tests

imvm committed:
Reverted local testing change

imvm committed:
Fixed ChromeDriver import

imvm committed:
Fixing Travis invalid variable bug

imvm committed:
Fixing comment

imvm committed:
Removed comment

Commit:
merge cjvg

Commit:
Merge branch 'cjvg-master'

Commit:
[#107523050] + Merge imvm

Commit:
Merge branch 'imvm-master'

Commit:
Update ImportConcepts.feature

Ignorando cenários por conta da incompatibilidade do Travis com o plug-in do Excel utilizado

Commit:
para o travis - 1 merge

Commit:
Merge pull request #8 from dansq/master

2a fase da entrega

Commit:
para o travis - 2 merge

Commit:
para o travis - 2 merge - corrigindo erros de compilacao do RegisterDiscipline.groovy

Commit:
para o travis - 2 merge - corrigindo erros de compilacao do RegisterDiscipline.groovy

renatodms committed:
Merge pull request #4 from gaabs/master

Entrega 2a iteração: 1 + 5 >> 7

Commit:
@ignore nos meus testes e modificações para compatibilizar com testes de contribuidores

Commit:
Merge branch 'master' of https://github.com/gaabs/TA

Conflicts:
	test/cucumber/ImportConcepts.feature

Commit:
Tentativa de compatibilizar testes da equipe

Commit:
Tentativa de compatibilizar testes da equipe

Commit:
Resolução de conflitos

Commit:
Resolução para merge

Commit:
Merge branch 'ehammo-master'

Commit:
Correção de conflitos remanescentes do merge.
Correção do GebConfig para rodar no Travis
Adição do método deleteAfterTest()

renatodms committed:
Merge pull request #5 from gaabs/master

Pull request com alterações

Commit:
Correção do método deleteAfterTest

Commit:
Correção de conflitos para segundo merge

Commit:
Merge branch 'renatodeyvson-master'

Commit:
Correção de conflitos pós merge.
